recipes/ipydatagrid/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "ipydatagrid" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ipydatagrid-{{ version }}.tar.gz
+  sha256: b7b387bf615d4513d1c24dc43c222a772a1f6789277e34995f5f8bd89291f128
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - {{ PYTHON }} -c 'import shutil as s, os; s.rmtree("examples"); os.unlink("pyproject.toml")'
+    - {{ PYTHON }} -m pip install --install-option="--skip-npm" . -vv
+
+requirements:
+  host:
+    - jupyter-packaging
+    - pip
+    - python >=3.6
+  run:
+    - bqplot >=0.11.6
+    - ipywidgets >=7.6.0,<8
+    - pandas >=0.25.0
+    - py2vega >=0.5.0
+    - python >=3.6
+
+test:
+  imports:
+    - ipydatagrid
+  commands:
+    - pip check
+    - jupyter labextension list 1>lab_extensions 2>&1
+    - cat lab_extensions | grep "ipydatagrid.*.*OK"
+  requires:
+    - pip
+    - jupyterlab >=3,<4
+    - m2-grep  # [win]
+
+about:
+  home: https://github.com/bloomberg/ipydatagrid
+  summary: Fast Datagrid widget for the Jupyter Notebook and JupyterLab
+  license: BSD-3-Clause
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - bollwyvl
```

Thanks! Would you mind adding the main contributors of the underlying package as well?
```suggestion
    - bollwyvl
    - ibdafna
    - mbektasbbg
    - kaiayoung
    - martinRenou
```

---

CC @ibdafna @mbektasbbg @kaiayoung @martinRenou - convention is that each of you need to accept the maintainer role by commenting here, otherwise we delete your name and then the responsibility is not on your plate anymore :).

---

I accept

---

I accept

---

I accept :)

---

cool, that's certainly enough folk to take care of a pure-python widget package!